src/generate-from-template.js

```diff
@@ -73,6 +76,9 @@ const generateFromTemplate = async ({ templateName }) => {
       templateValues,
     }
   ));
+  if (Object.keys(dynamicFolderNames).length > 0) {
+    renameFolders(path.resolve(`./${templateValues.projectName}`), { dynamicFolderNames });
```

Comment:
If we are parsing and creating the folders based on the new name, the new one will be created along with the old one because that still exists in the template so it will be created again. Thus opted for this approach.